cac-prod.yml

```diff
@@ -129,6 +129,12 @@ jenkins:
         name: ETHOS
         recurse: true
         title: ETHOS
+    - buildMonitor:
+        includeRegex: >-
+          ^HMCTS_.*(CMC).*\/(cmc-citizen-frontend)\/master
```

Comment:
why only citizen frontend?

Reply:
good point - we will be adding to all actually

Reply:
ill add others after we have citizen working as we want actually 👍